### ISSUE TITLE
The Walking Dead [PCSB00411]

### ISSUE THREAD

**Issue by Gerold55** [Vita3K/compatibility#1595]
# App summary
- App name: The Walking Dead
- App serial: PCSB00411
- App version: 1.00

# Vita3K summary
- Version: v0.1.8
- Build number: 3196
- Commit hash: https://github.com/vita3k/vita3k/commit/58a571e8
- CPU backend: Dynarmic
- GPU backend: OpenGL

# Test environment summary
- Tested by: Gerold55
- OS: Windows 11
- CPU: Intel Core i5 1600H
- GPU: NVIDIA GeForce GTX 1650
- RAM: 8 GB

# Issues
Some small graphical issues
![Screenshot (25)](https://github.com/Vita3K/compatibility/assets/13843095/d47d0d05-098c-4291-bccd-92ef2779e1a8)

# Screenshots

# Log

# Recommended labels
- ingame+

**Comment by yaboifungi** [comment]
Can someone test the game now?

**Comment by MrSythos** [comment]
yeah I assume this is also playable. I just tested the PCSE version about 2 or 3 months ago and that's playable.
